### PR TITLE
Feature/sequence io

### PR DESCRIPTION
**Pull request by temehi**
[NOTE] DO NOT MERGE BEFORE #77
------

An implementation of read functions for FASTA format.

Before I proceed with write() and other formats I would like to have comments on this one. That way you will have fewer lines to review

### CODE REVIEW

File: include/seqan3/io/sequence/sequence_file_format.hpp [hunk 1]
```diff
@@ -36,49 +36,47 @@
...
+                                                      std::ostream ostr)
 {
+    // static member
     t::file_extensions;
 
+    // member functions
     {
         v.read(dna4_string{},     // sequence
                std::string{},     // meta
-               std::string{},     // quality
```
Comment (smehringer) [inline include/seqan3/io/sequence/sequence_file_format.hpp]: what about reading the quality, why is this deleted?

**Comment by rrahn** [comment]
superseded by #225
